pennylane/plugins/default_qubit_autograd.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains an autograd implementation of the :class:`~.DefaultQubit`
+reference plugin.
+"""
+from pennylane.operation import DiagonalOperation
+
+try:
+    from pennylane import numpy as np
+
+except ImportError as e:
+    raise ImportError("default.qubit.autograd device requires Autograd")

```

Comment:
Yep, kept this in in case we ever move away from Autograd being a dependent of PL 😆 

We tend to be quite explicit with dependencies in the `pennylane/plugins` package, so I figured no harm in being explicit here.